docs/spark/spark-procedures.md

```diff
@@ -275,6 +275,11 @@ See the [`RewriteDataFiles` Javadoc](../../../javadoc/{{% icebergVersion %}}/org
 and <br/> [`SortStrategy` Javadoc](../../../javadoc/{{% icebergVersion %}}/org/apache/iceberg/actions/SortStrategy.html#field.summary)
 for list of all the supported options for this action.
 
+!!! Note
+    This procedure can also read the delete files created by merge-on-read mode if present
+    and rewrite to new data files by masking the result from delete files.
```

Comment:
Instead of "masking the result", thinking to keep more the language in https://iceberg.apache.org/spec/#version-2-row-level-deletes

 How about:
``` This procedure can also read applicable delete files to delete rows from the original data file during the rewrite.```